org.civicrm.bootstrapcivihr/scss/components/_loader.scss

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
this is not a component, and even if it was the name of the file would not reflect the name of the component

You are overriding a core screen style, create a `scss/overrides/crm` folder and place it in a aptly-named partial